test/controllers/auth/AuthActionSpec.scala

```diff
@@ -101,30 +101,26 @@ class AuthActionSpec extends FreeSpec with MustMatchers with MockitoSugar with O
     }
   }
 
+  implicit class Ops[A](a: A) {
```

Comment:
Move this to the top of the file
Rename the implicit class as Ops is ambiguous. Maybe `RetrievalOps`